lib/oat/adapters/siren.rb

```diff
@@ -91,7 +91,11 @@ class Field
           attr_reader :data
 
           def initialize(name)
-            @data = { :name => name }
+            @data = { :name => name, :class => []}
+          end
+
+          def class(value)
```

Comment:
I know, it's a little weird.  This is already being done in other parts of the Oat codebase though, for instance ```Oat::Adapters::Siren::Action#class``` .

I'd be fine if these became something like ```siren_class```.